chore: [IOBP-1319] lib upgrades

freddi301 opened the pull request:
## Short description
Upgrade 3 years old dependencies.

## List of changes proposed in this pull request


## How to test

LeleDallas commented on src/components/OperatorConvention/BucketCodeModal.tsx:
```diff
@@ -16,14 +16,18 @@ const BucketCodeModal = ({
-    );
+  const {
+    data,
+    isPending: isLoading,
```
Consider omitting the renaming and using `isPending` as-is for clarity

```suggestion
    isPending,
```

LeleDallas commented on src/components/OperatorActivations/OperatorActivations.tsx:
```diff
@@ -40,7 +40,7 @@ const OperatorActivations = () => {
   const {
     data: operators,
-    isLoading,
+    isPending: isLoading,
```
Consider omitting the renaming and using `isPending` as-is for clarity

```suggestion
    isPending,
```

LeleDallas commented on src/components/Form/EditActivationForm/EditActivationForm.tsx:
```diff
@@ -56,10 +56,13 @@ const CreateActivationForm = () => {
+  const {
+    data,
+    error,
+    isPending: isLoading
```
Consider omitting the renaming and using `isPending` as-is for clarity

```suggestion
    isPending
```

LeleDallas commented on src/components/Form/CreateActivationForm/CreateActivationForm.tsx:
```diff
@@ -21,7 +21,7 @@ const CreateActivationForm = () => {
   const { triggerTooltip } = useTooltip();
 
-  const { mutate, isLoading } =
+  const { mutate, isPending: isLoading } =
```
Consider omitting the renaming and using `isPending` as-is for clarity

```suggestion
  const { mutate, isPending } =
```

freddi301 commented:
All of the comments addressed in fba60127f58ac23a1fc1472cf23e1e3212eb8c13
I renamed all isLoading to isPending for consistency

Hantex9 commented:
@freddi301 Be aware that the build phase is failing: https://dev.azure.com/pagopaspa/cgn-onboarding-portal-projects/_build/results?buildId=219483&view=logs&j=c07a8676-a12c-5b19-50a4-38acac0f02e7&t=d3a7cc26-de72-513b-650b-2d038251fd58

freddi301 commented:
> @freddi301 Be aware that the build phase is failing: https://dev.azure.com/pagopaspa/cgn-onboarding-portal-projects/_build/results?buildId=219483&view=logs&j=c07a8676-a12c-5b19-50a4-38acac0f02e7&t=d3a7cc26-de72-513b-650b-2d038251fd58

discovered and waiting for upstream issue https://github.com/italia/design-react-kit/issues/1192

freddi301 commented:
> > @freddi301 Be aware that the build phase is failing: https://dev.azure.com/pagopaspa/cgn-onboarding-portal-projects/_build/results?buildId=219483&view=logs&j=c07a8676-a12c-5b19-50a4-38acac0f02e7&t=d3a7cc26-de72-513b-650b-2d038251fd58
> 
> discovered and waiting for upstream issue [italia/design-react-kit#1192](https://github.com/italia/design-react-kit/issues/1192)

fixed upstrea. removed patch

LeleDallas reviewed (commented):
With latest commit the app is not crashing!
I tried to run a build locally but the preview is not working...

<img width="1728" alt="Screenshot 2025-07-04 at 15 42 04" src="https://github.com/user-attachments/assets/aa4482f3-7213-4d98-afa1-e78a6ec5ecbe" />

freddi301 commented:
> With latest commit the app is not crashing! I tried to run a build locally but the preview is not working...
> 
> <img alt="Screenshot 2025-07-04 at 15 42 04" width="1728" src="https://private-user-images.githubusercontent.com/71103219/462515988-aa4482f3-7213-4d98-afa1-e78a6ec5ecbe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTE2NDY0NTIsIm5iZiI6MTc1MTY0NjE1MiwicGF0aCI6Ii83MTEwMzIxOS80NjI1MTU5ODgtYWE0NDgyZjMtNzIxMy00ZDk4LWFmYTEtZTc4YTZlYzVlY2JlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzA0VDE2MjIzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcxYjhhYmQxMmZjZTZhMWQ4YzYxMmQ2YmQ2MjQxY2I4YTc3ZmExMmFhOTVlOGI5ZGNkMjVkMGMyMmUxM2JlN2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.FJ3mDZ-0nD3wS0EsH2vKytkoy_E3-cbXRMzj2S4lQ-A">

fixed here d271d27e6d2deec4346656e510514934ddf6f050

LeleDallas reviewed (approved):
Looks good now!
